### DISCUSSION TITLE
Build issue

### DISCUSSION THREAD

**Discussion by robertgosz** [EdgeTX/edgetx#4294] [Q&A] [answered]
Hello everyone.
I'm trying to build EdgeTx for the first time using 2.9 branch under Ubuntu 22 looking at this guide: 
https://github.com/EdgeTX/edgetx/wiki/Build-Instructions-under-Ubuntu-20.04

I have used this command (Jumper T16 target):

_cmake -DPCB=X10 -DPCBREV=T16 -DDEFAULT_MODE=2 -DGVARS=YES -DPPM_UNIT=US -DLUA_MIXER=YES -DCMAKE_BUILD_TYPE=Release ../_

But then after issuing 'make configure' I've got an error: 

CMake Error at radio/src/CMakeLists.txt:567 (add_subdirectory):
  The source directory
    [...]/edgetx/radio/src/thirdparty/AccessDenied does not contain a CMakeLists.txt file.

I have checked and the AccessDenied directory is empty indeed. Any ideas what can be fishy here ?
Or it's not worth digging into that and I should rather use pre-built Docker image in order to build it ?

Best Regards
Robert

**Comment by pfeerick** [comment, marked as answer]
You probably missed the `--recursive` option when cloning the repo so that
submodules are also initalised. ;)

Run `git submodule update --init --recursive` to rectify that. If you have
any issues during the build itself, search in the issues, there is some
guidance onto updated dependency installs for newer versions of ubuntu.

**Reply by robertgosz** [reply]
You've nailed it @pfeerick 
Thank you very much!
